network/p2p.py

```
@@ -0,0 +1,34 @@
+    Currently implements a mock for simulation.
+    """
+    def __init__(self, handler_callback):
+        self.peers = []
+        self.handler_callback = handler_callback
+
+    async def start(self):
+        print("Network: Listening on /ip4/0.0.0.0/tcp/0")
+        # In real libp2p, we would await host.start() here
+
+    async def broadcast_transaction(self, tx):
+        msg = json.dumps({"type": "tx", "data": tx.to_dict()})
```

Ruff (0.15.0)

[error] Local variable `msg` is assigned to but never used

Remove assignment to unused variable `msg`

(F841)